packages/lodestar-cli/src/cmds/validator/options.ts

```diff
@@ -33,6 +34,13 @@ export const validatorOptions: CommandBuilder<{}, IValidatorCliOptions> = {
   force: {
     description: "Open validators even if there's a lockfile. Use with caution",
     type: "boolean"
-  } as Options
+  } as Options,
+
+  graffiti: {
+    description: "Specify your custom graffiti to be included in blocks (plain UTF8 text, 32 characters max)",
```

Comment:
But utf8 characters can be up to 4 bytes, right? 

Maybe expect ascii text? :smile:

Reply:
Truee

Reply:
Maybe we can be more flexible, convert to Buffer, then either pad or slice to 32 bytes.